scripts/deployment/globals_sepolia.json

```diff
@@ -0,0 +1 @@
+{"contractVerification":true,"useLedger":false,"derivationPath":"m/44'/60'/2'/0/0","providerName":"sepolia","dynamicContributionName":"Dynamic Contribution","dynamicContributionSymbol":"AUTONOLAS-DC","dynamicContributionBaseURI":"https://pfp.staging.autonolas.tech/","dynamicContributionAddress":"0x57f3703D281Fd7b3896969086Cf14Aa6Ace99d92","veOLASAddress":""}
```

Comment:
We need to deploy veOLAS on Sepolia at some point. However, tested the `DelegateContribute` deployment on sepolia with the veOLAS address from mainnet, and it was successfully verified.